pages/common/mocha.md

```diff
@@ -0,0 +1,15 @@
+# mocha
+
+> Execute Mocha JavaScript test runner
+
+- Run tests with default configuration, or as configured in `mocha.opts`
```

Comment:
Cool, thanks. I know it's nitpicky, but commas are hard to sneak out again afterwards and the world already has so many of them. Could you squash your commits together once you've gone over the comments?